lib/linguist/heuristics.yml

```
@@ -98,6 +98,12 @@ disambiguations:
   rules:
   - language: FreeBasic
     pattern: '^[ \t]*#(?:define|endif|endmacro|ifn?def|if|include|lang|macro)\s'
+- extensions: ['.bs']
+  rules:
+  - language: Bikeshed
+    pattern: '^(?i:<pre\s+class)\s*=\s*(|''|\"|\b)?metadata\1[^>\r\n]*>'
```

Typo:

```suggestion
    pattern: '^(?i:<pre\s+class)\s*=\s*(''|\"|\b)?metadata\1[^>\r\n]*>'
```

(Note that the `\b` capture is to match markup like `<pre class=metadata>` and not `<pre class=metadata_unrelated_tag>`).